src/ServicePulse.Host/app/js/app.http.js

```diff
@@ -2,15 +2,7 @@
     'use strict';
 
     function httpProvider($httpProvider) {
-
```

Comment:
Why remove these ?
Are these headers also used when loading views and other resources from ServicePulse itself ? I thought they were only used for ServiceControl REST api calls